app/routes/planet_routes.py

```diff
@@ -0,0 +1,113 @@
+from flask import Blueprint, abort, make_response, Response, request
+from app.models.planet import Planet
+from sqlalchemy import desc
+from ..db import db
+
+
+planets_bp = Blueprint("planets_bp", __name__, url_prefix="/planets")
+
+
+@planets_bp.get("")
+def get_all_planets():
+    query = db.select(Planet)
+
+    name_query = request.args.get("name")
+    if name_query:
+        query = query.where(Planet.name.ilike(f"%{name_query}%"))
+    
+    description_query = request.args.get("description")
+    if description_query:
+        query = query.where(Planet.description.ilike(f"%{description_query}%"))
+
+    diameter_query = request.args.get("diameter")
+    if diameter_query:
+        query = query.where(Planet.diameter.ilike(f"%{diameter_query}%"))
+
+    query = query.order_by(desc(Planet.diameter))
+    planets = db.session.scalars(query)
+
+    planet_response = []
+
+    for planet in planets:
+        planet_response.append(
+            {
+                "id": planet.id,
+                "name": planet.name,
+                "description": planet.description,
+                "diameter": planet.diameter
+            }
```

Comment:
Prefer your Planet model has a to_dict method so that it can be called in your different routes instead repeating code. It would also help your routes be more concise.